modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -17,6 +17,7 @@ The following table describes the CSI drivers that are installed with {product-t
 |CSI driver  |CSI volume snapshots  |CSI cloning  |CSI resize
 
 |AWS EBS (Tech Preview) | ✅ | - | ✅
+|Microsoft Azure (Tech Preview) | ??? | ??? | ??? 
```

Comment:
To match the name of the driver, either:
s/Microsoft Azure/Azure Disk/ or s/Microsoft Azure/Microsoft Azure Disk/
(If the latter, then move below Google for alpha ordering.)